### PR TITLE
Update Alphollon_C_Meriard.lua

### PR DESCRIPTION
**Pull request by Hinekri**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Correct two 119 Abju set trades.

## Steps to test these changes

NPC was looking for the wrong cursed items. May need a larger Audit only corrected the two I am aware of.

### CODE REVIEW

File: scripts/zones/Northern_San_dOria/npcs/Alphollon_C_Meriard.lua [hunk 1]
```diff
@@ -56,16 +56,16 @@ entity.onTrade = function(player, npc, trade)
...
-            8775, 27214, 27215, 27202, 27203,   8790, 27218, 27219, 27204, 27205,
-            8776, 27390, 27391, 27378, 27379,   8791, 27394, 27395, 27380, 27381,
-            8782, 26688, 26689, 26678, 26679,   9105, 25619, 25620, 25609, 25610,
-            8783, 26864, 26865, 26854, 26855,   9106, 25692, 25693, 25682, 25683,
-            8784, 27040, 27041, 20730, 20731,   9107, 27123, 27124, 27113, 27114,
-            8785, 27216, 27217, 27206, 27207,   9108, 27308, 27309, 27298, 27299,
-            8786, 27392, 27393, 27382, 27383,   9109, 27479, 27480, 27469, 27470,
+            8772, 26686, 26687, 26674, 26675,   8787, 26688, 26689, 26676, 26677,
+            8773, 26862, 26863, 26850, 26851,   8788, 26864, 26865, 26852, 26853,
+            8774, 27038, 27039, 20726, 20727,   8789, 27040, 27041, 20728, 20729,
```
Comment (neuromancerxi) [inline scripts/zones/Northern_San_dOria/npcs/Alphollon_C_Meriard.lua]: This should be:
```lua
            8774, 27038, 27039, 20726, 20727,   8789, 27040, 27041, 27028, 27029,
```

**Comment by WinterSolstice8** [comment]
wow, that table's magic numbers needs to get fixed, that's _quite_ monstrous

File: scripts/zones/Northern_San_dOria/npcs/Alphollon_C_Meriard.lua [hunk 2]
```diff
@@ -56,16 +56,16 @@ entity.onTrade = function(player, npc, trade)
...
+            8775, 27214, 27215, 27202, 27203,   8790, 27216, 27217, 27204, 27205,
+            8776, 27390, 27391, 27378, 27379,   8791, 27392, 27393, 27380, 27381,
+            8782, 26690, 26691, 26678, 26679,   9105, 25619, 25620, 25609, 25610,
+            8783, 26866, 26867, 26854, 26855,   9106, 25692, 25693, 25682, 25683,
+            8784, 27042, 27043, 20730, 20731,   9107, 27123, 27124, 27113, 27114,
```
Comment (neuromancerxi) [inline scripts/zones/Northern_San_dOria/npcs/Alphollon_C_Meriard.lua]: This should be:
```lua
            8784, 27042, 27043, 27030, 27031,   9107, 27123, 27124, 27113, 27114,
```

**Review by neuromancerxi — CHANGES_REQUESTED** [review]
Thanks for picking this up, looks like some typos made it in when I submitted this a couple years ago. Called out a couple other typos that can be fixed.

**Comment by neuromancerxi** [comment]
> wow, that table's magic numbers needs to get fixed, that's _quite_ monstrous

FWIW, this is reasonable to spot check manually with magic numbers:

```sql
SELECT * FROM item_basic
Where itemid in (
8784, 27042, 27043, 27030, 27031
);
```

Though I'll concede you could do roughly the same thing with:

```sql
SELECT * FROM item_basic
where name in (
"SHINRYU_ABJURATION_HANDS", 
"CARMINE_FINGER_GAUNTLETS", 
"CARMINE_FINGER_GAUNTLETS_+1", 
"BEWITCHED_FINGER_GAUNTLETS", 
"VOODOO_FINGER_GAUNTLETS"
);
```

Opinion: We'd want `scripts/globals/items.lua` and `sql/items_basic.sql` to agree on the name space for quicker validation. [Teo suggests](https://github.com/LandSandBoat/server/pull/1866#discussion_r889432952) that we're using `_P1` `_P2` instead of `+1` `+2`, but we'd want them to be in agreement to avoid any mix-ups.

**Comment by TeoTwawki** [comment]
I disagree on making the names match. item_basic.sql is meant to convey as closely as the limitations _file names_ will allow us the actual name shown in the client, whereas items.lua is meant to get as close as we can in what lua script will allow for a variable and string - these sadly have different constraints, but I wouldn't want to lose accuracy on one just to match the other. Fortunately, it's not too bad to perform a little conversion since there are unchanging conventions on what to convert'em to.

_p.s. FK the whole format of that lua table in that npc lol._ It needs broken up among other things.

Edit: that's not a request to redo it on this pr. Just to be clear.

**Comment by neuromancerxi** [comment]
> I disagree on making the names match. item_basic.sql is meant to convey as closely as the limitations _file names_ will allow us the actual name shown in the client, whereas items.lua is meant to get as close as we can in what lua script will allow for a variable and string - these sadly have different constraints, but I wouldn't want to lose accuracy on one just to match the other. Fortunately, it's not too bad to perform a little conversion since there are unchanging conventions on what to convert'em to.

That's fair. I'm looking at it from the perspective of validating large blocks like this. With magic numbers and 1:1 lua:sql, it's plug and play. With potentially arbitrary ENUMs the waters get a little murkier.

> Edit: that's not a request to redo it on this pr. Just to be clear.

Yep, same to my suggestions, I'm not suggesting a full rewrite of the PR.

File: scripts/zones/Northern_San_dOria/npcs/Alphollon_C_Meriard.lua (hunk 1 above)
Comment (Hinekri) [reply scripts/zones/Northern_San_dOria/npcs/Alphollon_C_Meriard.lua]: 8774, 27038, 27039, 20726, 20727, lol that ones wrong too 27026/27027, this NPC is a mess.

**Comment by Hinekri** [comment]
I just pushed it here because I made the small changes for us as the Abjurations are available via some Dynamic replacements for content not yet officially available. Not to worried if someone wants to take the PR to continue audit/reformat and it is open to edits by maintainers, do as you see fit ^^.

**Review by neuromancerxi — APPROVED** [review]

**Comment by zach2good** [comment]
Is this superseded by https://github.com/LandSandBoat/server/pull/2284?

**Comment by Hinekri** [comment]
> Is this superseded by #2284?

Yeah that took an extra step and probably cleared even more errors.

**Pull request closed without merging**